relay-profiling/tests/fixtures/profiles/sample_v2/valid.json

```diff
@@ -0,0 +1,64 @@
+{
+  "version": "2",
+  "chunk_id": "0432a0a4c25f4697bf9f0a2fcbe6a814",
+  "profiler_id": "4d229f1d3807421ba62a5f8bc295d836",
+  "timestamp": "2022-09-01T09:45:00.000Z",
+  "release": "0.1 (199)",
+  "platform": "cocoa",
+  "profile": {
+    "samples": [
+      {
+        "stack_id": 0,
+        "thread_id": "0x0000000102adc700",
```

Comment:
Shouldn't thread ids be numbers and not hex?

Reply:
That's the thing, it doesn't matter. For `cocoa`, we're focusing on queues and for the rest thread IDs. They're all `u64` anyway so need to be passed as strings. This is how we remove the `cocoa` specific `queue_metadata` field.

Having this as one string identifier simplifies things, you could have queue addresses or thread IDs in there.